workflow/python/ds-pack/1.1.0-base-python-3.10.13/README.md

```diff
@@ -0,0 +1,55 @@
+# workflow-python-ds-pack-1.1.0-base-python-3.10.13
+The workflow-python-ds-pack image contains a set of the most commonly used dependencies in addition to the essential packages that are required for running Peak Workflows.
+
+## Image details
+### Base image
+This image uses [python:3.8.13-slim-buster](https://hub.docker.com/layers/python/library/python/3.8.13-slim-buster/images/sha256-6258dcdb5fea7b710bfcfc3c889e022e4c6e9dd0ea962cfa73fbc130eff2c174?context=explore) as its base which is maintained by [the Docker Community](https://github.com/docker-library/python).
```

Comment:
This would need update as well

Reply:
Hey yeah readme updates were pending 😅 . Have updated them now.